kernel/Kconfig

```diff
@@ -549,6 +549,18 @@ config MAX_DOMAIN_PARTITIONS
 	help
 	  Configure the maximum number of partitions per memory domain.
 
+config USE_SWITCH
+	bool
+	prompt "Use new-style _arch_switch instead of __swap"
+	default n
+	help
+	The _arch_switch() API is a lower level context switching
+	primitive than the original __swap mechanism.  It is required
+	for an SMP-aware scheduler, or if the architecture does not
+	provide __swap.  In uniprocess situations where the
+	architecture provides both, _arch_switch incurs more somewhat
```

Comment:
" _arch_switch incurs more somewhat overhead and may be slower"

Nitpick: awkward English.
Suggest: "_arch_switch incurs somewhat more overhead."

Reply:
I... don't know why that ifdef is there anymore.  At a guess: because it used to hold an inline function that relied on other stuff under USE_SWTICH and is now just a vestigial function declaration.

Will fix that and the phrasing.